### PR TITLE
 Updated CopyColumns, DropColumns and SelectColumns samples.

### PR DESCRIPTION
**Pull request by zeahmed**
Related to #1209.

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CopyColumns.cs [hunk 1]
```diff
@@ -12,72 +12,60 @@ public static void Example()
             // as well as the source of randomness.
             var mlContext = new MLContext();
 
-            // Get a small dataset as an IEnumerable and them read it as ML.NET's data type.
-            IEnumerable<Microsoft.ML.SamplesUtils.DatasetUtils.SampleInfertData> data = Microsoft.ML.SamplesUtils.DatasetUtils.GetInfertData();
-            var trainData = mlContext.Data.LoadFromEnumerable(data);
+            // Create a small dataset as an IEnumerable.
+            var samples = new List<InputData>()
+            {
+                new InputData(){ ImageId = 1, Features = new [] { 1.0f, 1.0f, 1.0f} },
```
Comment (artidoro) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CopyColumns.cs]: >Features [](start = 46, length = 8)

You don't seem to need `Features`.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CopyColumns.cs [hunk 2]
```diff
@@ -12,72 +12,60 @@ public static void Example()
...
+            //  Label: 1 ImageId: 1
+            //  Label: 1 ImageId: 1
         }
 
-        private class OutputRow
+        private class InputData
         {
-            public float CustomValue { get; set; }
+            public int ImageId { get; set; }
+            public float[] Features { get; set; }
```
Comment (artidoro) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CopyColumns.cs]: >Features [](start = 27, length = 8)

Same here, I think you can drop it.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/DropColumns.cs [hunk 3]
```diff
@@ -12,71 +12,75 @@ public static void Example()
...
-            //  Age and Education were not available, so an exception was thrown: Could not find  column 'Age'.
+            //  ExtraColumn is not available, so an exception is thrown: Could not find  column 'ExtraColumn'.
             //  Parameter name: Schema
 
             // And we can write a few columns out to see that the rest of the data is still available.
-            var rowEnumerable = mlContext.Data.CreateEnumerable<SampleInfertDataTransformed>(transformedData, reuseRowObject: false);
+            var rowEnumerable = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, reuseRowObject: false);
             Console.WriteLine($"The columns we didn't drop are still available.");
             foreach (var row in rowEnumerable)
             {
```
Comment (artidoro) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/DropColumns.cs]: >{ [](start = 12, length = 1)

Please remove the brace, not needed for a one line loop.

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CopyColumns.cs [hunk 4]
```diff
@@ -12,72 +12,60 @@ public static void Example()
...
 
             // We can extract the newly created column as an IEnumerable of SampleInfertDataTransformed, the class we define below.
-            var rowEnumerable = mlContext.Data.CreateEnumerable<SampleInfertDataTransformed>(transformedData, reuseRowObject: false);
+            var rowEnumerable = mlContext.Data.CreateEnumerable<TransformedData>(transformedData, reuseRowObject: false);
 
             // And finally, we can write out the rows of the dataset, looking at the columns of interest.
-            Console.WriteLine($"Label, Parity, and CustomValue columns obtained post-transformation.");
+            Console.WriteLine($"Label and ImageId columns obtained post-transformation.");
             foreach (var row in rowEnumerable)
             {
```
Comment (artidoro) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CopyColumns.cs]: >{ [](start = 12, length = 1)

Please remove the braces as we don't need them for a one line loop.

**Comment by artidoro** [comment]
>             {

Please remove the braces as we don't need them for a one line loop.

---
Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/SelectColumns.cs:49 in 82859ca. [](commit_id = 82859ca888acb8d580397ce4a292eb30e7e9603a, deletion_comment = False)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CopyColumns.cs [hunk 5]
```diff
@@ -12,72 +12,60 @@ public static void Example()
...
-            // 39.0   1.0   0-5yrs      2.0         6.0       4.0      3.0  ...
-            // 34.0   1.0   0-5yrs      2.0         4.0       2.0      4.0  ...
-            // 35.0   1.0   6-11yrs     1.0         3.0       32.0     5.0  ...
+            // Convert training data to IDataView.
+            var dataview = mlContext.Data.LoadFromEnumerable(samples);
 
             // CopyColumns is commonly used to rename columns.
-            // For example, if you want to train towards Age, and your learner expects a "Label" column, you can
-            // use CopyColumns to rename Age to Label. Technically, the Age columns still exists, but it won't be
+            // For example, if you want to train towards ImageId, and your learner expects a "Label" column, you can
```
Comment (singlis) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CopyColumns.cs]: >learner [](start = 75, length = 7)

Should this be trainer?

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/DropColumns.cs [hunk 6]
```diff
@@ -12,71 +12,75 @@ public static void Example()
...
             }
 
             // Expected output:
             //  The columns we didn't drop are still available.
-            //  Case: 1 Induced: 1 Parity: 6
-            //  Case: 1 Induced: 1 Parity: 1
-            //  Case: 1 Induced: 2 Parity: 6
-            //  Case: 1 Induced: 2 Parity: 4
-            //  Case: 1 Induced: 1 Parity: 3
+            //  Age: 21 Geneder: Male Education: BS
```
Comment (singlis) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/DropColumns.cs]: Gender is mispelled

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/SelectColumns.cs [hunk 7]
```diff
@@ -11,41 +12,36 @@ public static void Example()
...
+                new InputData(){ Age = 27, Gender = "Female", Education = "PhD", ExtraColumn = 6 },
+            };
+
+            // Convert training data to IDataView.
+            var dataview = mlContext.Data.LoadFromEnumerable(samples);
 
             // Select a subset of columns to keep.
             var pipeline = mlContext.Transforms.SelectColumns("Age", "Education");
 
             // Now we can transform the data and look at the output to confirm the behavior of CopyColumns.
```
Comment (singlis) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/SelectColumns.cs]: SelectColumns

**Pull request merged**